extra/1-radio-stations.js

```diff
@@ -25,12 +33,20 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+const getStations=()=>{
```

Comment:
This code is technically correct, but I think we could improve the readability of the code. It's important to think about indentation and spacing here.